### PR TITLE
Add draft final mapping.

### PR DESCRIPTION
**Pull request by mlhale7**
## What does this Pull Request do?

This PR shares a starter document for our final deliverable. It also shares the style guide text we agreed upon.

## What's new?

The document is placed in the contents section since we'll deliver the final mapping there. I'm open to breaking this into multiple parts in the future so that it isn't so long, but for now I felt like we needed something shared to get started with.

## How should this be tested?

Are there any formatting issues? Is there any text or organizational decision you disagree with?

## Interested parties

@markpbaggett @CanOfBees @mathewjordan

### CODE REVIEW

**Review by markpbaggett — COMMENTED** [review]
@mlhale7 this looks good to me. 

One question:  do you want to add your updates to accessCondition from today's meeting before this is merged?

**Comment by mlhale7** [comment]
@markpbaggett - yes, that would be perfect. I've added the bakerav (restriction on access) and creative commons info. Let me know if anything doesn't make sense or would benefit from an addition.

File: docs/source/contents/4_draft_mapping.rst [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+be displayed to the user within the record, there is no reason to connect it with other accessCondition values. A note is
+sufficient for this use case.
+
+**Xpath**
+
+mods:accessCondition[@type="restriction on access"]
+
+**Decision**
+
+`Example record <https://digital.lib.utk.edu/collections/islandora/object/bakerav%3A10/datastream/MODS/view>`_
```
Comment (markpbaggett) [inline docs/source/contents/4_draft_mapping.rst]: @mlhale7, we'll talk about this later in greater detail, but to get that dumb warning to go away, you need to change your syntax to this:

```

`Example record <https://digital.lib.utk.edu/collections/islandora/object/bakerav%3A10/datastream/MODS/view>`__

```

**Review by markpbaggett — CHANGES_REQUESTED** [review]

File: docs/source/contents/4_draft_mapping.rst (hunk 1 above)
Comment (mlhale7) [reply docs/source/contents/4_draft_mapping.rst]: @markpbaggett - sorry, I didn't have Pygments appropriately installed. It looked like the links needed a distinct textual reference so I just changed my standard. Hopefully this fixes things.

**Review by markpbaggett — APPROVED** [review]

**Pull request merged**